### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by anask601**

### CODE REVIEW

File: tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,1420 @@
...
+.skills_bar {
+  background-color: var(--first-color-lighter);
+}
+
+.skills_percentage {
+  display: block;
+  background-color: var(--first-color);
+}
+
+.skills_html {
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css:550]:
```suggestion
.skills_html,
.skills_firebase,
.skills_figma {
```

File: tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,1420 @@
...
+  width: 90%;
+}
+
+.skills_css {
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css:554]:
```suggestion
.skills_css,
.skills_php {
```

File: tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,1420 @@
...
+
+.skills_python {
+  width: 55%;
+}
+
+.skills_figma {
+  width: 90%;
+}
+
+.skills_sketch {
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css:586]:
```suggestion
.skills_sketch,
.skills_photoshop {
```

File: tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,1420 @@
...
+  width: 60%;
+}
+
+.skills_react {
+  width: 85%;
+}
+
+.skills_php {
+  width: 80%;
+}
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css:568]: Remove this and similar others after making above change

File: tasks-by-users/anask601/01-js-portfolio/assets/js/main.js [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+  const scrollY = window.pageYOffset;
+
+  sections.forEach((current) => {
+    const sectionHeight = current.offsetHeight;
+    const sectionTop = current.offsetTop - 50;
+    sectionId = current.getAttribute("id");
+
+    if (scrollY > sectionTop && scrollY <= sectionTop + sectionHeight) {
+      document
+        .querySelector(".nav_menu a[href*=" + sectionId + "]")
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/assets/js/main.js:138]: Use template literal

File: tasks-by-users/anask601/01-js-portfolio/assets/js/main.js [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+        .classList.add("active-link");
+    } else {
+      document
+        .querySelector(".nav_menu a[href*=" + sectionId + "]")
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/assets/js/main.js:142]: Use template literal

File: tasks-by-users/anask601/01-js-portfolio/index.html [hunk 7]
```diff
@@ -0,0 +1,923 @@
...
+                            <span class="qualification_subtitle"> Figma - India</span>
+                            <div class="qualification_calender">
+                                <i class="uil uil-calendar-alt"></i>
+                                2017 -2019
+                            </div>
+                          </div>
+                          
+                          <div>
+                            <span class="qualification_rounder"></span>
+                            <!-- <span class="qualification_line"></span> -->
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/index.html:481]: remove comments

File: tasks-by-users/anask601/01-js-portfolio/index.html [hunk 8]
```diff
@@ -0,0 +1,923 @@
...
+
+                        </div>
+                        <!--==================== QUALIFICATION 4 ====================-->
+                        <div class="qualification_data">
+                          <div></div>
+
+                          <div>
+                            <span class="qualification_rounder"></span>
+
+                            <!-- <span class="qualification_line"></span> -->
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/index.html:419]: Remove comment

File: tasks-by-users/anask601/01-js-portfolio/index.html [hunk 9]
```diff
@@ -0,0 +1,923 @@
...
+        <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css"/>
+        <!--------------------------------------swiper css----------------------------------->
+        <link rel="stylesheet" href="assets/css/swiper-bundle.min.css" />
+        <!--------------------------------------css----------------------------------->
+        <link rel="stylesheet" type="text/css" href="assets/css/styles.css" />
+        <title>Portfolio</title>
+  </head>
+  <body>
+    <!----------------------------------------- HEADER ---------------------------------------------->
+      <header class="header" id="header">
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/01-js-portfolio/index.html:17]: For tags like header,footer etc you dont need to add class or id, you could directly access it with css selector like 

header {
}

footer {
}

**Comment by faran4engg** [comment]
Live demo link?

File: tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css (hunk 4 above)
Comment (anask601) [reply tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css]: Done @AlishaPrasad ma'am

File: tasks-by-users/anask601/01-js-portfolio/assets/js/main.js (hunk 5 above)
Comment (anask601) [reply tasks-by-users/anask601/01-js-portfolio/assets/js/main.js]: > Portfolio live link
https://anask601.github.io/portfolio/

**Comment by anask601** [comment]
> Live demo link?

https://anask601.github.io/portfolio/

File: tasks-by-users/anask601/01-js-portfolio/index.html (hunk 9 above)
Comment (anask601) [reply tasks-by-users/anask601/01-js-portfolio/index.html]: okay

File: tasks-by-users/anask601/01-js-portfolio/assets/js/main.js (hunk 5 above)
Comment (anask601) [reply tasks-by-users/anask601/01-js-portfolio/assets/js/main.js]: Done

**Review by faran4engg — APPROVED** [review]

**Review by coder-mezab — APPROVED** [review]

**Review by Abbas4tech — APPROVED** [review]

**Pull request merged**